invenio_banners/services/service.py

```diff
@@ -134,7 +115,14 @@ def update(self, identity, id, data):
 
         banner = self.record_cls.get(id)
 
-        self.record_cls.update(data, id)
+        # validate data
+        valid_data, errors = self.schema.load(
+            data,
+            context={"identity": identity},
```

Comment:
why is the context needed to deserialize the banner data?

Reply:
To check the permission on build context, see: https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/records/schema.py#L80